portfolio.html

```diff
@@ -43,133 +57,152 @@ <h1 class="item2">I'm a software developer</h1>
-    </div>
-
-   <div class="card-container">
-    <div class="card-1">
-      <img src="Img Plaholder@1x.png" alt="project image">
-    </div>
+        <div class="b_div">
+          <button class="see-button">
+            See Project
+          </button>
+        </div>
+      </div>
+    <!-- </div> -->
+
+    <!-- <div class="card-container"> -->
+      <div class='project-card'>
+        <div class="project-title">
+          <h3>
+            Multi-Post Stories Gain+Glory
+          </h3>
+        </div>
       
-    <div class="card-2">
-
-      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <ul class="project-abilites">
+          <li class="a1">Ruby on rails</li>
+          <li class="a1">css</li>
+          <li class="a1">JavScript</li>
+          <li class="a1">html</li>          
+        </ul>   
       
-      <ul class="abilites">
-        <li class="a1">Ruby on rails</li>
-        <li class="a1">css</li>
-        <li class="a1">JavScript</li>
-        <li class="a1">html</li>          
-      </ul>
-
-      <div class="b_div"><button class="button">See Project</button></div>
-    </div>
-
-   <div class="card-container">
-      <div class="card-1">
-        <img src="Img Plaholder@1x.png" alt="project image">
+        <div class="b_div">
+          <button class="see-button">
+            See Project
+          </button>
+        </div>
       </div>
-      
-      <div class="card-2">
-
-        <div class="title">
-          <h3>Multi-Post Stories Gain+Glory</h3>
+    <!-- </div> -->
+
+    <!-- <div class="card-container"> -->
+      <div class='project-card'>
+        <div class="project-title">
+          <h3>
+            Multi-Post Stories Gain+Glory
+          </h3>
         </div>
-        
-        <ul class="abilites">
+      
+        <ul class="project-abilites">
           <li class="a1">Ruby on rails</li>
           <li class="a1">css</li>
           <li class="a1">JavScript</li>
           <li class="a1">html</li>          
-        </ul>
-  
-        <div class="b_div"><button class="button">See Project</button></div>
+        </ul>   
+      
+        <div class="b_div">
+          <button class="see-button">
+            See Project
+          </button>
+        </div>
       </div>
+      <!-- </div> -->
+
+      <!-- <div class="card-container"> -->
+      <div class='project-card'>
+        <div class="project-title">
+          <h3>
+            Multi-Post Stories Gain+Glory
+          </h3>
+        </div>
+      
+        <ul class="project-abilites">
+          <li class="a1">Ruby on rails</li>
+          <li class="a1">css</li>
+          <li class="a1">JavScript</li>
+          <li class="a1">html</li>          
+        </ul>   
+      
+        <div class="b_div">
+          <button class="see-button">
+            See Project
+          </button>
+        </div>
       </div>
-
-   </section>
-   <section about_me_container>
```

Comment:
- [ ] Please, kindly fix the vertical spacing between these cards, to enable a very pixel-perfect design, like you'll be doing in your forthcoming job 😀.

- Your design
<img width="277" alt="wrong1" src="https://user-images.githubusercontent.com/41163571/221200100-6c1f3707-5e30-48df-951d-549e4dcabc1d.png">

- Figma design
<img width="251" alt="right1" src="https://user-images.githubusercontent.com/41163571/221200297-55210af6-a2ad-4166-9a50-58049078d4ad.png">